From 86c7d6d7663a7576efab8471b7c5ecc57f5a3e4f Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Wed, 18 Jun 2014 08:20:47 +0200
Subject: [PATCH 1/2] Start sddm.service after plymouth-quit.service

This will make sddm.service run after Plymouth retaining the splash.
See issue #196.
---
 services/sddm.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/sddm.service.in b/services/sddm.service.in
index f5c5f53..c476441 100644
--- a/services/sddm.service.in
+++ b/services/sddm.service.in
@@ -2,7 +2,7 @@
 Description=Simple Desktop Display Manager
 Documentation=man:sddm(1) man:sddm.conf(5)
 Conflicts=getty@tty1.service
-After=systemd-user-sessions.service getty@tty1.service
+After=systemd-user-sessions.service getty@tty1.service plymouth-quit.service
 
 [Service]
 ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/sddm
-- 
2.0.0

